Adds Amplitude font switch for examples (#4120)
jpmorganchase/salt-ds: site/docs/themes/index.mdx

When Amplitude is used, default heading font weight is set to medium (instead of semi bold for Open Sans), via `--salt-palette-text-heading-fontWeight` and `--salt-palette-text-display-fontWeight` variables.

> You'll need to install Amplitude font to your application (download from [internal site](https://go/salt-ds-internal/design/fonts) if you're part of JPMC) to make sure every user will see the font correctly.

Reviewer: ```suggestion
To ensure that every user sees the Amplitude font correctly, you'll need to add it to your application, which can be downloaded from the [fonts page](https://go/salt-ds-internal/design/fonts) on the internal site if you're part of JPMC.
```
